catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSourceTest.java

```diff
@@ -262,7 +275,6 @@ public static void setupClass() {
   }
 
   public void setUp(
```

Comment:
✏️ May be a good idea to change this to a private method, because
1. It *can* be private 🤷‍♂.
2. It may help to signify that this isn't the usual `setUp` method that junit runs before each test.

Reply:
The same thing occurs in `/src/test/java/org/codice/ddf/spatial/ogc/wfs/v1_0_0/catalog/source/WfsSourceTest.java`, so if you make this change be sure to make the corresponding one there.

Reply:
Done.